src/main/java/racingcar/domain/car/DefaultCar.java

```diff
@@ -0,0 +1,38 @@
+package racingcar.domain.car;
+
+import racingcar.domain.numbergenerator.RandomNumberGenerator;
+
+public class DefaultCar implements Car{
+    private static final int MOVE_THRESHOLD = 4;
+
+    private final RandomNumberGenerator numberGenerator;
+    private final String name;
+    private int position;
+
+    public DefaultCar(RandomNumberGenerator numberGenerator, String name) {
+        this.numberGenerator = numberGenerator;
+        this.name = name;
+        this.position = 0;
```

Comment:
생성자에서 초기 값을 지정하는 경우 주 생성자, 부생성자 방식을 추천드려요.

주 생성자는 private로 정의해도 되겠네요.

```suggestion
        this(name, 0);
```